src/frame.rs

```diff
@@ -1,24 +1,24 @@
-use crate::parser::{Expression, Statement};
-use std::collections::HashMap;
+use crate::parser::{Expression, FunctionExpression, Statement};
+use std::{collections::HashMap, rc::Rc};
 
-#[derive(Clone)]
-pub enum Value {
+pub(crate) enum Value<'a> {
```

Comment:
```suggestion
pub enum Value<'a> {
```

and similar every time

Reply:
Why do we make things wide open like this? Happy to do so but don't know why.

Reply:
Fixed in follow-up PR.